README.md

```diff
@@ -1,105 +1,168 @@
-		 '443.https.get.title',
-		 '443.https.tls.certificate.parsed.subject_dn',
-		 '443.https.tls.certificate.parsed.names',
-		 '443.https.tls.certificate.parsed.subject.common_name',
-		 '443.https.tls.certificate.parsed.extensions.subject_alt_name.dns_names',
-		 '25.smtp.starttls.tls.certificate.parsed.names',
-		 '25.smtp.starttls.tls.certificate.parsed.subject_dn',
-		 '110.pop3.starttls.tls.certificate.parsed.names',
-		 '110.pop3.starttls.tls.certificate.parsed.subject_dn']
-
-data = list(c.search("80.http.get.headers.server: Apache AND location.country: Japan", 
-                             IPV4_FIELDS, max_records=10))		 
-print (data)
+IPV4_FIELDS = [
+    "ip",
+    "updated_at",
+    "80.http.get.title",
+    "443.https.get.title",
+    "443.https.tls.certificate.parsed.subject_dn",
+    "443.https.tls.certificate.parsed.names",
+    "443.https.tls.certificate.parsed.subject.common_name",
+    "443.https.tls.certificate.parsed.extensions.subject_alt_name.dns_names",
+    "25.smtp.starttls.tls.certificate.parsed.names",
+    "25.smtp.starttls.tls.certificate.parsed.subject_dn",
+    "110.pop3.starttls.tls.certificate.parsed.names",
+    "110.pop3.starttls.tls.certificate.parsed.subject_dn",
+]
+
+data = list(
+    c.search(
+        "80.http.get.headers.server: Apache AND location.country: Japan",
+        IPV4_FIELDS,
+        max_records=10,
+    )
+)
+print(data)
 ```
 
-Data API
---------
-
-The Data API allows programatic access to the raw data files.
+Here's how to use the Website index.
 
 ```python
-import censys.data
+import censys.websites
+
+c = censys.websites.CensysWebsites(api_id="XXX", api_secret="XXX")
+
+# the report method constructs a report using a query, an aggregation field, and the
+# number of buckets to bin
+c.report(
+    """ "welcome to" AND tags.raw: "http" """,
+    field="80.http.get.headers.server.raw",
+    buckets=5,
+)
 
-c = censys.data.CensysData(api_id="XXX", api_secret="XXX")
+# the view method lets you see the full JSON for an website
+c.view("google.com")
+
+# the search method lets you search the index using indexed fields, full text, and
+# combined predicates
+for result in c.search(
+    "80.http.get.headers.server: Apache", max_records=10
+):
+    print(result)
+
+# you can optionally specify which fields you want to come back for search results
+WEBSITE_FIELDS = [
+    "ip",
+    "domain",
+    "updated_at",
+    "80.http.get.title",
+    "443.https.get.title",
+    "443.https.tls.certificate.parsed.subject_dn",
+    "443.https.tls.certificate.parsed.names",
+    "443.https.tls.certificate.parsed.subject.common_name",
+    "443.https.tls.certificate.parsed.extensions.subject_alt_name.dns_names",
+]
+
+data = list(
+    c.search(
+        "80.http.get.headers.server: Apache",
+        WEBSITE_FIELDS,
+        max_records=10,
+    )
+)
+print(data)
+```
+
+## Developing
+
+Install dev dependencies with:
+
+```bash
+pip install -e .[dev]
+```
 
-series_name = '22-ssh-banner-full_ipv4'
```

Comment:
This code still works, we just aren't producing new scan specific datasets. Not sure if we should keep it

Reply:
I believe this format for series name was replaces by one including date.

Reply:
Added in [`README.md#107`](https://github.com/censys/censys-python/blob/eaff32d828bfc606a713293964858b6836dffae6/README.md#data)